### ISSUE TITLE
No labels in repo but the label still shows up in the label picker

### ISSUE THREAD

**Issue by codemanmeet** [HubTurbo/HubTurbo#1036]
![screen shot 2015-10-09 at 3 32 45 pm](https://cloud.githubusercontent.com/assets/4116346/10388288/2fbf6f9c-6e9b-11e5-93f7-4fbb48a31640.png)

I deleted all the labels from a repo. 'Enhancement' was applied from before for that repo. Label picker still shows that 'enhancement' can be applied.

**Comment by acruis** [comment]
Did you do an F5 refresh before opening the label picker?

On Fri, Oct 9, 2015 at 3:35 PM, Manmeet Saluja <notifications@github.com>
wrote:

> [image: screen shot 2015-10-09 at 3 32 45 pm]
> <https://cloud.githubusercontent.com/assets/4116346/10388288/2fbf6f9c-6e9b-11e5-93f7-4fbb48a31640.png>
>
> I deleted all the labels from a repo. 'Enhancement' was applied from
> before for that repo. Label picker still shows that 'enhancement' can be
> applied.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/HubTurbo/HubTurbo/issues/1036>.
>



-- 
*Nguyen Quoc Dat*
*Year 3, Computing (CS), NUS*
*Prince George's Park Residences*
*(+65)**947-15125*

**Comment by codemanmeet** [comment]
Yup I did press F5! Label still shows up in the label picker dialog.

**Comment by dariusf** [comment]
Check if the ETag for the repo is the latest, and if the label appears in the cache?

**Comment by codemanmeet** [comment]
![screen shot 2015-10-10 at 2 41 06 am](https://cloud.githubusercontent.com/assets/4116346/10402523/784a3190-6ef8-11e5-9c45-3b8d2bb062ea.png)

![screen shot 2015-10-10 at 2 02 32 am](https://cloud.githubusercontent.com/assets/4116346/10402527/7e7b77cc-6ef8-11e5-8ee2-737a527367ff.png)

Yup, the ETag is the latest and the label appears in the cache which it shouldn't.
